packages/file/src/file.ts

```diff
@@ -0,0 +1,125 @@
+import path from 'path'
+import StreamZip from 'node-stream-zip'
+import FileFormat from '@sketch-hq/sketch-file-format-ts'
+import Zip from 'adm-zip'
+
+/**
+ * Given a path to a Sketch file on the file system, this function unzips the
+ * JSON entries and parses them out into a SketchFile object.
+ */
```

Comment:
Maybe good opportunity to revisit the comment to provide better documentation:

```suggestion
/**
 * Returns a {@link Promise} that resolves with a fully parsed {@link SketchFile}
 * object once the file at the given path has been unzipped and read successfully.
 *
 * @param filepath -  The file path to the Sketch document
 */
```